lib/engine/game/g_18_eus/game.rb

```diff
@@ -374,6 +376,23 @@ def setup_privates
 
           @companies = privates.values.sort.map { |v| v.first(4) }.flatten
         end
+
+        def bidbox_privates
+          @companies.select { |c| (!c.owner || c.owner == @bank) && !c.closed? }.first(self.class::BIDDING_BOX_PRIVATE_COUNT)
+        end
+
+        def setup_bidboxes
+          bidbox_privates.each { |c| c.owner = @bank }
```

Comment:
Why not do this at setup time?

Reply:
bidbox_privates only contains those privates that are currently in a bidbox, not all of them.  Only those currently up for bid can be bank owned, because the view displays all companies that are bank owned.